src/main/java/frc/robot/commands/AlignByAprilTagGyro.java

```diff
@@ -63,6 +66,7 @@ public void initialize() {
     @Override
     public void execute() {
         if (swerveDrive.getCorrectRot()) {
+            // Why are you making a new pose from `odometry.getPose()`? Why not just use `odometry.getPose()`?
```

Comment:
Yeah good point, that was a carry over from when we needed to alter odometry.getPose() now we should just use odometry.getPose()